gpt4all-chat/modellist.cpp

```diff
@@ -1175,7 +1175,7 @@ void ModelList::updateModelsFromDirectory()
 
                 // All files that end with .bin and have 'ggml' somewhere in the name
                 if (((filename.endsWith(".bin") || filename.endsWith(".gguf")) && (/*filename.contains("ggml") ||*/ filename.contains("gguf")) && !filename.startsWith("incomplete"))
-                    || (filename.endsWith(".txt") && (filename.startsWith("chatgpt-") || filename.startsWith("nomic-")))) {
```

Comment:
I think this makes the || part for nomic irrelevant as the nomic remote embedding model also ends with .txt.

Reply:
fixed